content/blog/2014/05/2014-05-05-evolve-os-pre-alpha-released-or-shaking-off-the-vapours/index.md

```diff
@@ -55,7 +55,7 @@ info to find things to play with)
 
 ### **Reporting Issues**
 
-Please leave issues in the [Evolve OS Google+ Community](https://plus.google.com/communities/103032596316713958671) for now, as we currently have nowhere to 
```

Comment:
I'm not 100% sure about removing this kind of information. Google+ is known to be dead, and completely the URL prevents one from perhaps checking an archived copy in the Internet Wayback Machine or other archives. It also still resolves to *something* at least, and you can get the information about the Google+ deprecation on that page.